client/ayon_core/pipeline/create/creator_plugins.py

```diff
@@ -185,20 +191,20 @@ class BaseCreator:
 
     # Instance attribute definitions that can be changed per instance
     # - returns list of attribute definitions from
-    #       `ayon_core.pipeline.attribute_definitions`
-    instance_attr_defs = []
+    #       `ayon_core.lib.attribute_definitions`
+    instance_attr_defs: "list[AbstractAttrDef]" = []
```

Comment:
Shouldn't this use `List` for older pythons?
```suggestion
    instance_attr_defs: List['AbstractAttrDef"] = []
```

Reply:
Not if it's within a string, right? ;) That's why I did that for the full thing. It'd only be loaded for type checking then.